<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Week 9 - QDialog, QTabWidget &amp; MenuBar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_inputNama">
     <attribute name="title">
      <string>Input Nama</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_inputNama">
         <property name="text">
          <string>Input Nama</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_nama">
         <property name="text">
          <string>Nama : </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_pilihFont">
     <attribute name="title">
      <string>Pilih Font</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_pilihFont">
         <property name="text">
          <string>Pilih Font</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pilihFont">
         <property name="text">
          <string>Nama : </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_bukaFile">
     <attribute name="title">
      <string>Buka File</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_bukaFile">
         <property name="text">
          <string>Buka File .txt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_bukaFile">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuFitur">
    <property name="title">
     <string>Fitur</string>
    </property>
    <addaction name="actionInput_Nama"/>
    <addaction name="actionPilih_Font"/>
    <addaction name="actionBuka_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFitur"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionKeluar">
   <property name="text">
    <string>Keluar</string>
   </property>
  </action>
  <action name="actionInput_Nama">
   <property name="text">
    <string>Input Nama</string>
   </property>
  </action>
  <action name="actionPilih_Font">
   <property name="text">
    <string>Pilih Font</string>
   </property>
  </action>
  <action name="actionBuka_File">
   <property name="text">
    <string>Buka File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
